modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java

```diff
@@ -65,30 +87,29 @@ protected static String addNewImportsJSPHeader(
 
 			return StringUtil.replaceFirst(
 				newContent, jspHeader,
-				getNewImportsJSPHeader(new String[] {jspHeader}, newImports));
+				getNewImportsJSPHeader(jspHeader, newImports));
 		}
 
-		return getNewImportsJSPHeader(new String[0], newImports) + newContent;
+		return getNewImportsJSPHeader("", newImports) + newContent;
 	}
 
 	protected static String getNewImportsJSPHeader(
-		String[] jspHeaders, String[] newImports) {
+		String lastJSPHeader, String[] newImports) {
 
 		StringBundler sb = new StringBundler(4);
 
-		for (String jspHeader : jspHeaders) {
-			sb.append(jspHeader);
+		if (!lastJSPHeader.isEmpty()) {
```

Comment:
@nicolas-sss, can we use `Validator.isNotNull()` instead?

I think this is important to prevent `NullPointerExceptions`. Imagine a scenario that a developer used `null` as the first parameter instead of `""`. :wink: